### ISSUE TITLE
Directory structure and organization

### ISSUE THREAD

**Issue by mollyfish** [mollyfish/advanced-presentation-AT-MK-wip#74]
Break main.scss apart so all it contains is `@import` tags.

**Comment by toalina** [comment]
We could brainstorm here, and that might help dictate additional issues!

So far, we have: 
- icons
- reset
- nav
- header_deco

What else?

**Comment by mollyfish** [comment]
Check out the `figure_refactor` branch - I started pulled lots of stuff out of `main.scss`, but we will probably want to reorganize

**Comment by toalina** [comment]
Yup! That looks good so far. I think the current class discussion confirms how you pulled out all that stuff. 

As the discussion goes too, I wonder if we should consider putting all the @mixin into a _mixin.scss... But that might overkill. Nervous!

**Comment by mollyfish** [comment]
As Dale said, as long as we are consistent with our system, any system is okay.  :)
